lib/exceptions.h

```diff
@@ -76,10 +84,17 @@ class MYSQLPP_EXPORT Exception : public std::exception
 	}
 
 	/// \brief Create exception object
-	Exception(const std::string& w) throw() :
+#if __cplusplus >= 201103L
+	Exception(std::string w) throw() :
+	what_(std::move(w))
+	{
+	}
+#else
+    Exception(const std::string& w) throw() :
```

Comment:
Changes the parameter needlessly. You can argue about whether pass-by-value is better, but you don't get to change it based solely on build-time configuration options. The place for that argument is in libtabula.